ansible/roles/deploy-baremetal/templates/kickstart

```diff
@@ -0,0 +1,44 @@
+# Created by Ansible on {{ansible_date_time.date}} at {{ansible_date_time.time}}.
```

Comment:
Ok, but first tell me what is the criteria for using .j2:

```
$ ls roles/*/templates
roles/builds/templates:
github_ssh_config.j2

roles/jenkins-master/templates:
admin_config.xml.j2  jenkins.conf  jenkinsd

roles/jenkins-seed-job/templates:
seed_job_config.xml.j2

roles/nginx/templates:
nginx.conf

roles/systemd/templates:
jenkins.service
```
At first I took a quick look and thought we we're not using the extension.